[core]/esx_multicharacter/server/commands.lua

```diff
@@ -1,28 +1,16 @@
 ESX.RegisterCommand('setslots', 'admin', function(xPlayer, args, showError)
-	local slots = MySQL.scalar('SELECT `slots` FROM `multicharacter_slots` WHERE identifier = ?', {
-		args.identifier
-	})
-
-	if not slots then
-		MySQL.update('INSERT INTO `multicharacter_slots` (`identifier`, `slots`) VALUES (?, ?)', {
+		MySQL.insert('INSERT INTO `multicharacter_slots` (`identifier`, `slots`) VALUES (?, ?) ON DUPLICATE KEY UPDATE `slots` = VALUES(`slots`)', {
```

Comment:
Why remove whole code?

Reply:
Because duplicate entries are managed inside the mysql insert function, it's not needed anymore.

Reply:
Yes it's okey but you can't seprate now add and update notify. You removed slotsedit notify. This is not correct because the player does not know if he has just added a slot or if he has already modified an existing slot.

BUT
If we don't look at this in the future we will have to delete the slotsedit key from all locales and yes this will be the right direction thanks again.

Reply:
The command is literally 'setslots': you set the slots you want with a given number, what's the point of knowing if you are adding or updating a slot?

Reply:
@0xDEMXN Hi! Yes nothing, please delete slotsedit keys all locales and i can merge. Thank you.